### DISCUSSION TITLE
How do I connect to the created instance?

### DISCUSSION THREAD

**Discussion by miskin1702** [HotNoob/Oracle-Free-Arm-VPS-PS#8] [Q&A]
I used the script, I haven't gotten results yet, but it will be late when I get results. How do I connect to the created instance? ssh? Or can you explain another method? I'm very newbie :)

**Comment by HotNoob** [comment]
https://github.com/HotNoob/Oracle-Free-Arm-VPS-PS/issues/3#issuecomment-1989307038
